.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -124,7 +124,36 @@ stages:
       displayName: Generate Matrices
       name: generate_variables_step
 
-- stage: build_windows
+- stage: build_windows_tracer
```

Comment:
As we've renamed this task, we'll need to do two things before we can merge this PR:

- Update the new no-op pipeline.yml to rename `build_windows` to `build_windows_tracer`.
- Mark the `build_windows_tracer` task as required in GitHub (instead of the `build_windows` task)

Reply:
I've updated noop yml pipeline in e7c81415a45abbbac69f4297a95e88eb99739387
Still need to mark `build_windows_tracer` as required

Reply:
I made `build_windows` no longer required so it doesn't block this PR.